### DISCUSSION TITLE
Nav Bar option

### DISCUSSION THREAD

**Discussion by ghost** [twbs/bootstrap#39165] [Q&A]
So that I don't have to adjust the navigation menu on all web pages every time when I make a change, can I create one leading "bootstrap nav bar Webpage" and then I link it into my webpages?
If it is possible, how to proceed and is there an example on the Bootstrap website?

**Comment by patkon** [comment]
[PHP](https://www.php.net) is usually used for this kind of magic.

Here is a fairly simple example of how to include elements:
https://www.w3schools.com/php/php_includes.asp

**Reply by ghost** [reply]
Thanks for the help.
In the meantime I also found a "js-script-method" and for the php-variant I had to activate the IIS.
But both work as desired, this is very helpful for me.
